### PR TITLE
Moving ATs from the at file to individual patches

### PR DESCRIPTION
**Pull request by Machine-Maker**
waiting for paperweight 1.3.10

### CODE REVIEW

**Comment by MiniDigger** [comment]
should prolly add a message ontop of the AT file how ATs are handled now and update contributing.md, but this looks great

**Comment by NoahvdAa** [comment]
The contribution guide currently doesn't mention ATs at all, so I feel like documenting them should belong in a separate PR.

**Comment by MiniDigger** [comment]
oh. also checked the mcdev imports file and that doesnt mention much about it works either, so prolly fine.
I still think a line in the paper at file about "hey, you prolly want to put ATs in the patch header" would be cool.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -236,6 +236,32 @@ There are exceptions, especially in Spigot-related files
...
+to change the visibility or remove the final modifier from fields, methods, and classes. Inside the `build-data/paper.at`
+file, you can add ATs that are applied when you `./gradlew applyPatches`. You can read about the format of ATs 
+[here](https://mcforge.readthedocs.io/en/latest/advanced/accesstransformers/#access-modifiers).
+
+### Important
+ATs should be included in the patch file which requires them within the commit message. Do not commit any changes to the
+`build-data/paper.at` file, just use it to initially change the visibility of members until you have finalized what you 
+need. Then, in the commit message for the patch which requires the ATs, add a header at the bottom of the commit message
+before any co-authors. It should look like the following after you `./gradlew rebuildPatches`.
+```
```
Comment (jpenilla) [inline CONTRIBUTING.md:251]: should include a co author in the example

**Review by jpenilla — APPROVED** [review]

**Review by Owen1212055 — COMMENTED** [review]
Might be nice to say you can use your IDE to get methods signatures for the ATs. 
For example, 
```java
private ChunkAccess doFill(Blender blender, StructureManager structureAccessor, RandomState noiseConfig, ChunkAccess chunk, int minimumCellY, int cellHeight) {
```
View -> Show Bytecode

```
 // access flags 0x2
  private doFill(Lnet/minecraft/world/level/levelgen/blending/Blender;Lnet/minecraft/world/level/StructureManager;Lnet/minecraft/world/level/levelgen/RandomState;Lnet/minecraft/world/level/chunk/ChunkAccess;II)Lnet/minecraft/world/level/chunk/ChunkAccess;
   L0
   ```

**Comment by jpenilla** [comment]
Maybe want to add
```kotlin
tasks.collectAtsFromPatches {
    // Uncomment while updating for a new Minecraft version
    // extraPatchDir.set(layout.projectDirectory.dir("patches/unapplied/server"))
}
```
somewhere towards the bottom of the root build file

**Review by jpenilla — APPROVED** [review]

**Comment by Machine-Maker** [comment]
> Might be nice to say you can use your IDE to get methods signatures for the ATs. For example,
> 
> ```java
> private ChunkAccess doFill(Blender blender, StructureManager structureAccessor, RandomState noiseConfig, ChunkAccess chunk, int minimumCellY, int cellHeight) {
> ```
> 
>  
> View -> Show Bytecode
> 
> ```
>  // access flags 0x2
>   private doFill(Lnet/minecraft/world/level/levelgen/blending/Blender;Lnet/minecraft/world/level/StructureManager;Lnet/minecraft/world/level/levelgen/RandomState;Lnet/minecraft/world/level/chunk/ChunkAccess;II)Lnet/minecraft/world/level/chunk/ChunkAccess;
>    L0
> ```

I don't really think we want to get into teaching how to read bytecode. The forge docs cover everything we need. Plus, that view > bytecode doesn't work like half the time

**Review by Owen1212055 — APPROVED** [review]

**Review by MiniDigger — APPROVED** [review]
🥳🎉

**Pull request merged**